api/capi/src/nnstreamer-capi-single.c

```diff
@@ -73,6 +74,16 @@ ml_single_open (ml_single_h * single, const char *model,
   /* init null */
   *single = NULL;
 
+  if (FALSE == gst_init_check (NULL, NULL, &err)) {
```

Comment:
Try ```gboolean gst_is_initialized ()``` before calling ```gst_init_check```.

If it is already initialized, don't initialize again.


We actually need to add ``` gst_is_initialized()``` in pipeline API implementation as well.

Reply:
`gst_init_check()` function internally checks whether initialization is done or not. So in my opinion, we don't need to check it by calling `gst_is_initialized()`. This mechanism is already applied to `Tizen`.

* https://git.tizen.org/cgit/platform/upstream/gstreamer/tree/gst/gst.c#n341
```cpp
gboolean
gst_init_check (int *argc, char **argv[], GError ** err)
{
  ...
  g_mutex_lock (&init_lock);

  if (gst_initialized) {
    GST_DEBUG ("already initialized gst");
    g_mutex_unlock (&init_lock);
    return TRUE;
  }
  ...
```

Reply:
Oh. Good!